Edit description of Glossika

ML-Chen opened the pull request:
I wasn't able to do the build, so I edited the index.html and resources.md files manually. Here's the error that I was getting from WSL after install jq and yq:
```
build.sh -- Writing to '/mnt/c/Users/micha/Documents/Git/Resource-Guide/index.html'
./build.sh: 114: printf: Illegal option - 
yq: Error running jq: ParserError: expected '<document start>', but found '<block sequence start>'
  in "<stdin>", line 718, column 1.
jq: error (at <stdin>:1): Cannot index object with number
build.sh -- Writing to '/mnt/c/Users/micha/Documents/Git/Resource-Guide/resources.md'
```

Aryailia commented:
I added `%s` to the line 114 so it should work now.

If you're not familiar with POSIX shellscript, the interpreter is basically `printf '- '` as `printf -` and hyphens are the convention for specifying arguments to programs (or built-in commands).

Rebase to master and update your pull request if possible.

ML-Chen committed:
Update description of Glossika
Update description of Glossika
Update description of Glossika

Update README.md

Finish updating index.html

ML-Chen commented:
Awesome, the command works for me in WSL now. I rebased to master and ran the build script.

Aryailia commented:
Thanks for the help!
